docs/cody/core-concepts/context.mdx

```diff
@@ -51,7 +84,7 @@ Cody's usage and limits policies help optimize its performance and ensure cost-e
 
 ### Cody context window size
 
-Cody's context window represents the amount of contextual information provided to assist in generating code or responses. Currently, the context window size for both chat and commands is set to **7000 tokens per interaction**.
+Cody's context window represents the amount of contextual information provided to assist in generating code or responses. Currently, the context window size for both chat and commands is set to **8000 tokens per interaction**.
```

Comment:
This can vary based on the model being used, I don't know that we need to specify

Reply:
Removing it until we get some consolidated numbers.